### PR TITLE
solana-cli: format strings

### PR DESCRIPTION
**Pull request by CriesofCarrots**
This PR is spun out from https://github.com/solana-labs/solana/pull/34274

#### Problem
Long string literals can cause rustfmt to fail, which results in rustfmt failing to format entire functions. There are many instances of this in the solana-cli.

#### Summary of Changes
Format and wrap strings to allow rustfmt to work again.
This PR was initiated by adding `format_strings = true` to the workspace rustfmt.toml. In general, this did a nice job.

However, I chose to manually preserve or improve some of the original strings where the formatting seemed to break readability. (It is just my own personal preference that `[default: ..]` blocks not be broken across multiple lines.)
If we impose `format_strings` on the workspace in the future, these strings will be "wrong" and need fixing.

### CODE REVIEW

File: cli/src/address_lookup_table.rs [hunk 1]
```diff
@@ -113,15 +121,18 @@ impl AddressLookupTableSubCommands for App<'_, '_> {
...
                         .arg(
                             Arg::with_name("authority")
                                 .long("authority")
                                 .value_name("AUTHORITY_SIGNER")
                                 .takes_value(true)
                                 .validator(is_valid_signer)
-                                .help("Lookup table authority [default: the default configured keypair]")
+                                .help(
+                                    "Lookup table authority [default: the default configured \
+                                     keypair]",
```
Comment (steviez) [inline cli/src/address_lookup_table.rs:134]: Nit: In some of the other ones, you put
```
[default: the default configured keypair]
```
on its' own line. Altho the first line `Lookup table authority` is pretty bare so I'm fine with the wrap too

File: cli/src/clap_app.rs [hunk 2]
```diff
@@ -67,16 +67,19 @@ pub fn get_clap_app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> A
...
+                    "root",         // Deprecated as of v1.5.5
+                    "max",          // Deprecated as of v1.5.5
                 ])
                 .value_name("COMMITMENT_LEVEL")
                 .hide_possible_values(true)
                 .global(true)
-                .help("Return information at the selected commitment level [possible values: processed, confirmed, finalized]"),
+                .help(
+                    "Return information at the selected commitment level [possible values: \
+                     processed, confirmed, finalized]",
```
Comment (steviez) [inline cli/src/clap_app.rs:81]: nit: Put this on its' own line ?
```
[possible values: processed, confirmed, finalized]",
```
Was going to ask whey we didn't utilize `.possible_values()` but then I scrolled up in the diff 😅

File: cli/src/validator_info.rs [hunk 3]
```diff
@@ -223,9 +227,12 @@ impl ValidatorInfoSubCommands for App<'_, '_> {
                                 .value_name("PUBKEY")
                                 .takes_value(true)
                                 .validator(is_pubkey)
-                                .help("The pubkey of the Validator info account; without this argument, returns all"),
+                                .help(
+                                    "The pubkey of the Validator info account; without this \
+                                     argument, returns all",
```
Comment (steviez) [inline cli/src/validator_info.rs:232]: nit - I think what is being returned is implied, but sounds a little off reading it out loud. What do you think ?
```
argument, returns all Validator info accounts",
```

File: cli/src/stake.rs [hunk 4]
```diff
@@ -153,52 +156,62 @@ impl StakeSubCommands for App<'_, '_> {
...
                 )
-                .arg(
-                    pubkey!(Arg::with_name("custodian")
+                .arg(pubkey!(
+                    Arg::with_name("custodian")
                         .long("custodian")
                         .value_name("PUBKEY"),
-                        "Authority to modify lockups. ")
-                )
+                    "Authority to modify lockups. "
```
Comment (steviez) [inline cli/src/stake.rs:167]: The trailing period with space after it doesn't quite seem to line up with what the macro drops in.
https://github.com/solana-labs/solana/blob/d84dcd37bcb6afceb533a3883ec05e048a190903/cli/src/lib.rs#L2-L11

We seemingly don't need the trailing space or the period. Ie, running `solana create-stake-account --help`, I see:
```
        --custodian <PUBKEY>
            Authority to modify lockups. , one of:
              * a base58-encoded public key
              * a path to a keypair file
              * a hyphen; signals a JSON-encoded keypair on stdin
              * the 'ASK' keyword; to recover a keypair via its seed phrase
              * a hardware wallet keypair URL (i.e. usb://ledger)
```
There are a bunch of these so thinking we defer to another PR, but figured I'd mention it as I just saw it. Probably worth auditing all instances of `pubkey!(` to check this too

**Review by steviez — COMMENTED** [review]
Just a few nits

File: cli/src/address_lookup_table.rs (hunk 1 above)
Comment (CriesofCarrots) [reply cli/src/address_lookup_table.rs:134]: Dang it! Thanks for catching these.

File: cli/src/clap_app.rs (hunk 2 above)
Comment (CriesofCarrots) [reply cli/src/clap_app.rs:81]: Haha, yeah... 😢

File: cli/src/validator_info.rs (hunk 3 above)
Comment (CriesofCarrots) [reply cli/src/validator_info.rs:232]: I like it!

**Comment by CriesofCarrots** [comment]
Sorry @steviez , I squashed the nit fixes into their relevant file-commits before I realized that would make it hard to re-review 😞

File: cli/src/cluster_query.rs [hunk 5]
```diff
@@ -131,114 +134,117 @@ impl ClusterQuerySubCommands for App<'_, '_> {
...
-            .alias("get-genesis-hash")
+            SubCommand::with_name("leader-schedule")
+                .about("Display leader schedule")
+                .arg(
+                    Arg::with_name("epoch")
+                        .long("epoch")
+                        .takes_value(true)
+                        .value_name("EPOCH")
+                        .validator(is_epoch)
+                        .help("Epoch to show leader schedule for. [default: current]"),
```
Comment (steviez) [inline cli/src/cluster_query.rs:192]: nit: Don't think we've done a period after last word and before `[default: ...]`

File: cli/src/cluster_query.rs [hunk 6]
```diff
@@ -316,13 +322,16 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                     Arg::with_name("slot_limit")
                         .long("slot-limit")
                         .takes_value(true)
-                        .help("Limit results to this many slots from the end of the epoch [default: full epoch]"),
+                        .help(
+                            "Limit results to this many slots from the end of the epoch [default: \
```
Comment (steviez) [inline cli/src/cluster_query.rs:326]: nit: Move entire `[default: ...]` to own line?

File: cli/src/cluster_query.rs [hunk 7]
```diff
@@ -394,33 +403,35 @@ impl ClusterQuerySubCommands for App<'_, '_> {
...
                         .validator(is_slot)
-                        .help(
-                            concatcp!(
-                                "Minimum slot distance from the tip to consider a validator delinquent. [default: ",
-                                DELINQUENT_VALIDATOR_SLOT_DISTANCE,
-                                "]",
-                        ))
+                        .help(concatcp!(
+                            "Minimum slot distance from the tip to consider a validator \
+                             delinquent. [default: ",
```
Comment (steviez) [inline cli/src/cluster_query.rs:416]: nit: Drop the trailing period to make it `delinquent [default: ...` ?

**Review by steviez — COMMENTED** [review]
> I squashed the nit fixes into their relevant file-commits before I realized that would make it hard to re-review

No worries, luckily not every file was touched and I had bene using the feature of marking files as viewed. I think I found a few more that we might as well grab

**Review by steviez — APPROVED** [review]
LGTM!

**Pull request merged**